File: cpp/BMI.cxx
/*
 * BMI.cpp
 * 
 */


#include <iostream>
#include <math.h>
using namespace std;



int main(int argc, char **argv)

{
    float b, m, w;
    m = w = b =0;
	cout << "Podaj swoją masę" << endl;
    cin >> m;
    cout << "Podaj swój wzrost w metrach" << endl;
    cin >> w;
    b = m /(w*w);
    cout << "Twoje BMI  wynosi: " << b << endl;

    if (b < 18.5){
        cout << "NIEDOWAGA" << endl;
        }
    else if ((b > 18.5) && (b < 24.9)){
        cout << "NORMA" << endl;
        }
    else if ((b >= 25) && (b < 30)){
        cout << "NADWAGA" << endl;
        }
    else if (b >= 30){
        cout << "OTYŁOŚĆ" << endl;
        }
	return 0;
}

File: Python/BMI.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-


def main(args):
    m = float(input("Podaj swoją masę: "))
    print(m)
    w = float(input("Podaj swój wzrost w metrach: "))
    print(w)

    b = m / (w * w)
    print ("Twoje BMI wynosi: ", b)

    if b < 18.5:
        print ("niedowaga")
    elif b >= 24.9:
        print("norma")
    elif b >= 25:
        print("nadwaga")
    elif b >= 30:
        print("otyłość")

    return 0


if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))
